nodejs/eks/examples/tests/dupe-tags/index.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016-2019, Pulumi Corporation.
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+import * as eks from "@pulumi/eks";
+import * as pulumi from "@pulumi/pulumi";
+import * as iam from "./iam";
+
+const projectName = pulumi.getProject();
+
+// Create a vpc with non-default config.
+const vpc = new awsx.ec2.Vpc(`${projectName}-vpc`);
+
+// Create an IAM role and instanceProfile to use with the Cluster & NodeGroup.
+const role = iam.createRole(`${projectName}-role`);
+const instanceProfile = new aws.iam.InstanceProfile(`${projectName}-instanceProfile`, {role: role});
+
+/*
+ * Create an EKS Cluster with non-default config and duplicate tags for the
+ * supported resource tags to test that *at most* one of the dupe tags
+ * gets set for the respective resource.
+ *
+ * Since all taggable resources inherit cluster.tags, we will set all
+ * of the dupes in there, as well as in the targeted resource, to validate that
+ * the tag is only set once.
+ */
+const cluster = new eks.Cluster(`${projectName}`, {
+    vpcId: vpc.id,
+    subnetIds: vpc.publicSubnetIds,
+    deployDashboard: false,
+    instanceRole: role,
+    tags: {
+        "project": "foo",
+        "org": "bar",
+        "autoScalingGroupTagTest": "true",
+        "clusterSecurityGroupTagTest": "true",
+        "nodeSecurityGroupTagTest": "true",
+        "cloudFormationTagTest": "true",
+    },
+    clusterSecurityGroupTags: {
+        "myClusterSecurityGroupTag": "true",
+        "clusterSecurityGroupTagTest": "false",
+    },
+    nodeSecurityGroupTags: {
+        "myNodeSecurityGroupTag": "true",
+        "nodeSecurityGroupTagTest": "false",
+    },
+});
+
+// Create a NodeGroup in cluster.
+const ng1 = new eks.NodeGroup(`${projectName}-nodegroup1`, {
```

Comment:
If you aren't using the variables for anything, you can omit them.
```suggestion
new eks.NodeGroup(`${projectName}-nodegroup1`, {
```